README.md

```diff
@@ -109,3 +71,41 @@ The following files will be created:
 - `index.html`: start page for the project
 - `pyscript.toml`: project metadata and config file
 - `main.py`: a "Hello world" python starter module
+
+#### Use --wrap to embed a python file OR a command string
+
+- ##### Embed a Python script into a PyScript HTML file
+
+```shell
+$ pyscript create --wrap <filename.py>
+```
+
+This will generate a file called `<filename.html>` by default.
+This can be overwritten with the `-o` or `--output` option:
+
+```shell
+$ pyscript create --wrap <filename.py> -o <another_filename.html>
+```
+
+- ##### Open the script inside the browser using the `--show` option
+
+```shell
+$ pyscript create --wrap <filename.py> --show
+```
+
+- ##### Set a title for the browser tab
+
+You can set the title of the browser tab with the `--title` option:
+
+```shell
+$ pyscript create --wrap <filename.py> --title "My cool app!"
+```
```

Comment:
In theory we were to use the `name` field in the toml to set the page title. Is thi doing something different? I think it'd be good to have a bit of feature discovery discussion about it to try and find the best option

Reply:
Removed `title` as well along with `show` as of now.

For the `wrap` option, either a python file or a command string, the default title is `PyScript App` as of now.